lib/feedx/producer.rb

```diff
@@ -41,9 +44,9 @@ def perform
         nil
       end if local_rev.positive?
 
-      @stream.create metadata: { META_LAST_MODIFIED => local_rev.to_s } do |fmt|
+      @stream.create metadata: { META_LAST_MODIFIED => local_rev.to_s }, **@opts do |fmt|
```

Comment:
basically, options that are passed to `Feedx::Producer.new` are then propagated to:

* `Feed::Stream#create`
  * `BFS::Blob#create`
  * `Feedx::Compression::Abstract#writer`
  * `Feedx::Format::Abstract#encoder`